packages/core/core-flows/src/product/workflows/export-products.ts

```diff
@@ -1,18 +1,49 @@
-import { WorkflowData, createWorkflow } from "@medusajs/workflows-sdk"
-import { useRemoteQueryStep } from "../../common"
+import {
+  WorkflowData,
+  createWorkflow,
+  transform,
+} from "@medusajs/workflows-sdk"
 import { WorkflowTypes } from "@medusajs/types"
+import { generateProductCsvStep, getAllProductsStep } from "../steps"
+import { useRemoteQueryStep } from "../../common"
+import { sendNotificationsStep } from "../../notification"
 
 export const exportProductsWorkflowId = "export-products"
 export const exportProductsWorkflow = createWorkflow(
   exportProductsWorkflowId,
   (
     input: WorkflowData<WorkflowTypes.ProductWorkflow.ExportProductsDTO>
   ): WorkflowData<void> => {
-    const products = useRemoteQueryStep({
-      entry_point: "product",
-      fields: input.select,
-      variables: input.filter,
-      list: true,
+    const products = getAllProductsStep(input)
```

Comment:
question: Not sure if this is very relevant now, but did you consider a long-running workflow for the export?

Reply:
Ah yes indeed, I already was thinking about it in that way (which is why I was waiting for the notification in the tests), but didn't make it to run in the background. Let me change it.

Reply:
@olivermrbl done!